packages/rspack-test-tools/tests/hotCases/css/recovery/__snapshots__/web/2.snap.txt

```diff
@@ -42,41 +42,29 @@ self["webpackHotUpdate"]('main', {
   no-console,
   func-names
 */
-/** @typedef {any} TODO */
-const normalizeUrl = __webpack_require__(/*! ./normalize-url */ "../../../../../rspack/dist/builtin-plugin/css-extract/hmr/normalize-url.js");
+const normalizeUrlFn = __webpack_require__(/*! ./normalize-url */ "../../../../../rspack/dist/builtin-plugin/css-extract/hmr/normalize-url.js");
```

Comment:
could we keep this variable name `normalizeUrl` ?

Reply:
Javascript is incredibly annoying...

![image](https://github.com/user-attachments/assets/5f072038-0154-4b2b-a017-637e0bd17958)
![image](https://github.com/user-attachments/assets/b28649e2-0f23-4587-bff0-5ff16d87a75c)